### PR TITLE
Add basic header component

### PR DESCRIPTION
**Pull request by kaytcat**

### CODE REVIEW

File: packages/kununu-header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Header from './Header';
+import HeaderNav from './HeaderNav';
+import HeaderNavItem from './HeaderNavItem';
+
+Header.Header = Header;
+Header.HeaderNav = HeaderNav;
+Header.HeaderNavItem = HeaderNavItem;
+
+module.exports = Header;
```
Comment (karlhorky) [inline packages/kununu-header/index.jsx]: Can all of this be done with [the `export {} from` syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export)?

```js
export {Header} from './Header';
export {HeaderNav} from './HeaderNav';
export {HeaderNavItem} from './HeaderNavItem';
```

Comment (kaytcat) [inline packages/kununu-header/index.jsx]: yes it can! I meant to do this.. oops

File: packages/client-gear-playground/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "client-gear-playground",
+  "version": "1.0.0",
+  "description": "A place to test and development kununu components",
```
Comment (feychou) [inline packages/client-gear-playground/package.json]: A place to test and develop client-gear? It seems the playground will not be just for components after all.

File: packages/kununu-header/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "description": "kununu Header",
+  "main": "index.jsx",
+  "author": "kununu",
+  "license": "ISC",
+  "devDependencies": {
+    "react": "15.4.2"
+  },
+  "dependencies": {
+    "kununu-logo": "1.0.0",
+    "kununu-theme": "^3.0.0"
```
Comment (feychou) [inline packages/kununu-header/package.json]: I think this should be in the peer deps too.

File: packages/kununu-logo/package.json [hunk 4]
```diff
@@ -12,6 +12,7 @@
     "react": ">=15.0.0"
   },
   "dependencies": {
+    "kununu-theme": "^3.0.0",
```
Comment (feychou) [inline packages/kununu-logo/package.json]: Here too, this should probably be in the peerDeps. Which means we can just add it to the devDependencies.

**Comment by feychou** [comment]
LGMT except `kununu-theme` should be just installed in the hosting app in my opinion.

**Pull request merged**